jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -332,7 +332,38 @@ def _uncertainty(result):
         for function in FUNCTIONS:
             # TODO: update specutils to allow ALL analysis to take regions and continuum so we
             # don't need these if statements
-            if function == "Equivalent Width":
+            if function == "Line Flux":
+                meta = self.dataset.selected_dc_item.meta
+                telescope = meta.get('telescope', meta.get('TELESCOP', ''))
+                if telescope == 'JWST':
```

Comment:
Are these assumptions always true for *any* JWST data? If not, we need to filter more by supported data categories.

https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/index.html

Reply:
The original request by @PatrickOgle did not specify a subset of JWST data, so I assume "any" JWST data is correct? But maybe we can get confirmation here?

Reply:
Most of the level 3 JWST science products will be either in flux units of `MJy` or `Jy`, or surface brightness units of `MJy/sr`.  For the L3 spectral data products, it's only the s2d and s3d products that are in units of surface brightness, and even then it's only some instruments.

Reply:
Sounds like as good a confirmation as we're gonna get; thanks Brian!